utils/cache.py

```
@@ -15,21 +15,34 @@
 from rest_framework import status
 from rest_framework.authtoken.models import Token
 from rest_framework.response import Response
+from zmq import has
```

Ruff (0.11.9)

`zmq.has` imported but unused

Remove unused import: `zmq.has`

(F401)